Filters/OrdinalDateFilter.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Fluid;
+using Fluid.Values;
+using OrchardCore.Liquid;
+using System;
+using System.Threading.Tasks;
+
+namespace Etch.OrchardCore.Liquid.Filters
+{
+    public class OrdinalDateFilter : ILiquidFilter
+    {
+        public ValueTask<FluidValue> ProcessAsync(FluidValue input, FilterArguments arguments, TemplateContext ctx)
+        {
+            if (input.IsNil())
+            {
+                return new ValueTask<FluidValue>(new StringValue("Invalid Date passed to ordinal filter"));
+            }
+
+            var date = (DateTime)input.ToObjectValue();
+
+            var dayString = string.Empty;
+
+            if (date.Day == 1 || date.Day == 21 || date.Day == 31)
+            {
+                dayString = "\\s\\t";
```

Comment:
Out of curiosity, why are the back slashes needed?

Reply:
Need to escape the characters, else they'll be passed into the `DateTime.Format` without them, and it'll think they're part of the formatting, instead of just a string we want displayed

My example before them was giving me December 10P11 2020, as `th` turns into A/P and then the hour, and the article was posted at 11PM, doing this displays it as December 10th 2020